source/includes/fact-oplog-background.rst

```diff
@@ -1,7 +1,15 @@
-The :term:`oplog` in the source cluster must be large enough to track
-events that happen during the time it takes to complete the initial
-sync to the destination cluster.
+
+``mongosync`` applies operations in the ``oplog`` on the source cluster
+to the data on the destination cluster.  When operations 
+that ``mongosync`` has not applied roll off the ``oplog`` 
+on the source cluster, the sync fails and ``mongosync`` exits.
+
+During the initial sync, ``mongosync`` may apply operations at a slower
+rate. After ``mongosync`` completes the initial sync, it applies changes 
+faster and stays more current in the source cluster ``oplog``.
 
 If you anticipate syncing a large data set, or if you plan to pause
-synchronization for an extended period of time, increase the size of the
-replica set ``oplog`` in the source cluster.
+synchronization for an extended period of time, you might exceed the
+:term:`oplog window`. Use the :setting:`~replication.oplogSizeMB` setting
+to increase the size of the ``oplog`` on the source cluster.
```

Comment:
Should we include some of the original text that we reverted for the 1.4.0 release with [DOCSP-30835](https://jira.mongodb.org/browse/DOCSP-30835)? From the way this is currently phrased, I think it wouldn't be totally clear to me that this is a new feature or that it's a (substantial) improvement over 1.4.0.

A couple points on mongosync maintaining a recent position in the oplog and mitigating the risk of the sync failing would be a good inclusion IMO, but let me know what you think.

Reply:
Fixed